benchmark/v5.2-performance-benchmarking-with-tpch.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: TiDB TPC-H 100GB Performance Test Report -- v5.1 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
-summary: Compare the TPC-H 100GB performance of TiDB 5.1 MPP mode, Greenplum 6.15.0, and Apache Spark 3.1.1.
+title: TiDB TPC-H 100GB Performance Test Report -- v5.2 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
```

Comment:
The alias is added in https://github.com/pingcap/docs/pull/6169/files#diff-00428752dcd3c9acf5fe78f74e52359b2e723aeb5d03c91b87170bcdfd545469R4.